nmatrix.gemspec

```diff
@@ -56,7 +56,7 @@ EOF
   gem.required_ruby_version = '>= 1.9'
 
   gem.add_dependency 'packable', '~> 1.3', '>= 1.3.5'
-  gem.add_development_dependency 'bundler', '~>1.6'
+  gem.add_development_dependency 'bundler', '>=1.6'
```

Comment:
I think you want to try this: 

https://github.com/SciRuby/daru-view/pull/131/commits/d6de2718707bce40f4b2d9a21888c65c3e27f422